hddm/models/base.py

```diff
@@ -703,8 +703,8 @@ def __init__(self, data, bias=False, include=(),
         if bias:
             self.include.add('z')
 
-        possible_parameters = ('v', 'a', 't', 'z', 'st', 'sz', 'sv', 'p_outlier','dual_alpha')
```

Comment:
I think we need `dual_alpha` elsewhere, can you just add `alpha` here?

Reply:
It's actually not used anymore, instead a parameter called pos_alpha is directly created in the HDDMrl and Hrl classes if dual=True.